etna/whatson/models.py

```diff
@@ -829,3 +834,314 @@ class EventFilterForm(forms.Form):
             attrs={"class": "filters__toggle-input", "data-js-family": ""}
         ),
     )
+
+
+class ExhibitionHighlights(Orderable):
```

Comment:
Question: should this be named `ExhibitionHighlight` (singular)? It looks like one instance of this class maps to a single "highlight".